config/management/operational/src/keys.rs

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use libra_management::{config::ConfigPath, error::Error, secure_backend::ValidatorBackend};
+use serde::Serialize;
+use std::{fs::File, io::Write, path::PathBuf};
+use structopt::StructOpt;
+
+#[derive(Debug, StructOpt)]
+struct ExtractKey {
+    #[structopt(flatten)]
+    config: ConfigPath,
+    /// The keys name in storage
+    #[structopt(long)]
+    key_name: String,
+    /// Location to store the key
+    #[structopt(long)]
+    key_file: PathBuf,
+    #[structopt(flatten)]
+    validator_backend: ValidatorBackend,
+}
+
+#[derive(Debug, StructOpt)]
+pub struct ExtractPublicKey {
+    #[structopt(flatten)]
+    extract_key: ExtractKey,
+}
+
+impl ExtractPublicKey {
+    pub fn execute(self) -> Result<(), Error> {
+        let config = self
+            .extract_key
+            .config
+            .load()?
+            .override_validator_backend(&self.extract_key.validator_backend.validator_backend)?;
+
+        let storage = config.validator_backend();
+        let key_name = Box::leak(self.extract_key.key_name.into_boxed_str());
```

Comment:
Curious why do we leak it?

Reply:
hahahaha, rust hackery, I should have commented, but this effectively let's us make it a &'static str from a String... Since this is a CLI app, the String value should have no issues living long enough.